src/libraries/System.DirectoryServices.Protocols/tests/LdapSessionOptionsTests.cs

```diff
@@ -14,7 +14,23 @@ public class LdapSessionOptionsTests
         [PlatformSpecific(TestPlatforms.Windows)]
         [InlineData(ReferralChasingOptions.None)]
         [InlineData(ReferralChasingOptions.External)]
-        public void ReferralChasing_Set_GetReturnsExpected(ReferralChasingOptions value)
+        public void ReferralChasing_Set_GetReturnsExpected_On_Windows(ReferralChasingOptions value)
```

Comment:
this test seems identical to the one below, can you join them together? `[PlatformSpecific(TestPlatforms.Windows | TestPlatforms.Linux)]` should work

Reply:
They have the same content, but different inputs, since setting `External` is not supported on Linux but it is on Windows. Can we scope the `InlineData` attribute for each platform? (Sorry if that's obvious: I didn't see examples of this elsewhere.)